src/ghga_service_commons/httpyexpect/server/exceptions.py

```diff
@@ -18,6 +18,8 @@
 
 from abc import ABC
 
+from pydantic import ConfigDict
+
 try:  # workaround for https://github.com/pydantic/pydantic/issues/5821
```

Comment:
This workaround should not be necessary any more, can you check?

Reply:
You are right, it seems to be fixed now.